### PR TITLE
init: implemented init for cartridge app

### PR DESCRIPTION
**Pull request by psergee**
Part of #117

### CODE REVIEW

File: cli/cmdcontext/cmdcontext.go [hunk 1]
```diff
@@ -195,3 +197,9 @@ type RpmDebCtx struct {
 	// DepsFile is a path to a file of dependencies.
 	DepsFile string
 }
+
+type InitCtx struct {
+	// SkipConfig - if set, disables cartridge & tarantoolctl config analysis,
+    // so init does not try to get directories information from exitsting config files.
```
Comment (0x501D) [inline cli/cmdcontext/cmdcontext.go:203]: Fix indentation.

**Review by 0x501D — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,3 +30,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Module ``tt build``, to build an application.
 - Module ``tt install``, to install tarantool/tt.
 - Module ``tt remove``, to remove tarantool/tt.
+- Module ``tt init``, to create tt environment configuration file.
```
Comment (LeonidVas) [inline CHANGELOG.md:33]: No, it must to be added to the new release.

File: cli/cmdcontext/cmdcontext.go [hunk 3]
```diff
@@ -98,13 +100,13 @@ type RunningCtx struct {
 // ConnectCtx contains information for connecting to the instance.
 type ConnectCtx struct {
 	// Username of the tarantool user.
-	Username    string
+	Username string
```
Comment (LeonidVas) [inline cli/cmdcontext/cmdcontext.go:103]: Not related with the commit. (And I really don't understand what you're fixing here)
Here and bellow.

File: cli/cmdcontext/cmdcontext.go [hunk 4]
```diff
@@ -195,3 +197,9 @@ type RpmDebCtx struct {
 	// DepsFile is a path to a file of dependencies.
 	DepsFile string
 }
+
+type InitCtx struct {
```
Comment (LeonidVas) [inline cli/cmdcontext/cmdcontext.go:201]: Add description of the context.

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patchset.
Commit title:
`init: implement init module...` -> `tt: implement init module ...`
Generally ok. See several comments bellow.

File: cli/cmdcontext/cmdcontext.go (hunk 3 above)
Comment (psergee) [reply cli/cmdcontext/cmdcontext.go]: This is done automatically by go fmt.

**Review by LeonidVas — APPROVED** [review]
LGTM

**Pull request merged**